packages/react-native/src/primitives/Checkbox/types.ts

```diff
@@ -1,14 +1,14 @@
 import { StyleProp, ViewStyle } from 'react-native';
 
-import { IconButtonProps } from '@aws-amplify/ui-react-native/src/primitives/IconButton';
+import { IconButtonProps } from '../IconButton';
 
 import { LabelProps } from '../Label';
 
 export interface CheckboxProps<T>
-  extends Omit<IconButtonProps, 'source' | 'style'> {
+  extends Omit<IconButtonProps, 'source' | 'style' | 'accessibilityRole'> {
```

Comment:
Realize this is a pattern that we started doing earlier but not sure we should be omitting `accessibilityRole`. If we are planning on preventing users from overriding, it should be done on the base component props we are extending from, but ultimately I think we should allow users to define their own values for `accessibilityRole` and providing sensible defaults. Thoughts?

Reply:
I agree with providing sensible defaults, but also allowing users to define their own values for `accessibilityRole`. 

Implementation detail: If we were to allow overrides, should we spread the `rest` props after our 
```jsx
    <View>
      accessibilityRole="checkbox"
      {...rest}
    </View>
```

Reply:
Refactored!